### PR TITLE
Updated project policies for 2023

### PR DESCRIPTION
**Pull request by borjasotomayor**
Here are the updated project policies for 2023. They are effectively the same as the last time the class was offered, but with a bunch of updates related to the switch from chiventure to ChiGame. Besides looking out for chiventure or C programming remnants, we are also on time to tweak the grading scheme, etc. for the project.

### CODE REVIEW

File: website/project/issue-pr-grading.rst [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+should be documented on the GitHub issue. For example:
+
+-  In Task #1, suppose you implement a barebones `Circle` class with
+   a constructor but no other methods. You could commit and push that
+   code, and post a comment in the issue.
+-  In Task #3, you could post a comment every time you finish reviewing
+   an individual library (instead of just posting everything together at
+   the end)
+
+Pro tip: If you include an issue number in a commit message, then it
```
Comment (elizabethli31) [inline website/project/issue-pr-grading.rst:203]: Give an example of a commit message?

File: website/project/issue-pr-grading.rst [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+will decide when to merge that code into ``main``).
+
+Because the ``dev`` branch should contain stable code, pull requests to
+``dev`` must be carefully reviewed before they are approved. If you want
+to merge code into ``dev``, your code must be reviewed by (1) someone in
+your team and (2) your senior developer. Please note that the senior
+developer will only do their code review once your teammate has completed
+their review. Additionally, when your code affects code
+owned by another team, you must have an additional reviewer from each
+team whose code is affected.
```
Comment (elizabethli31) [inline website/project/issue-pr-grading.rst:261]: Would both reviewers from the teams get points? And do two TAs need to be on the review?

File: website/project/issue-pr-grading.rst [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+an issue or a pull request may be determined to be a "minor"
+issue or pull request (e.g., if you create a pull request to fix
+a small bug that only requires changing a few lines of code).
+
+Based on this, you will receive the following number of points
+for a completed issue or pull request:
+
+.. cssclass:: table-bordered
+
++---+-----------------+-----------------+
```
Comment (elizabethli31) [inline website/project/issue-pr-grading.rst:305]: If an issue has an associated pull request, will students receive points for both? Ex. earning an E on the issue and PR will give 30 points. If so, I think this should be clarified

File: website/project/issue-pr-grading.rst [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+Like a task, a pull request should have a descriptive title, a
+descriptive summary, updates on the progress of the pull request
+(especially since changes may need to be done based on the code
+reviews), and a closing statement. The assignees for the pull request
+must be everyone who contributed to the code that is being merged.
+
+Before a pull request is approved, the merge being merged must be
+"up to date" with ``dev``, meaning that any changes that happened
+in ``dev`` while the pull request was being reviewed must be merged
+into the pull request's branch. It is sometimes not uncommon for
```
Comment (elizabethli31) [inline website/project/issue-pr-grading.rst:279]: Is there any type of etiquette we want to include for making branches / naming them?

File: website/project/issue-pr-grading.rst [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+| S | 5       | 1     | 10      | 2     |
++---+---------+-------+---------+-------+
+
+Please note that earning an N or U does not accrue points.
+
+On an issue, all assignees will receive the same amount of points
+for a completed issue. If a pull request is approved, all assignees
+*and* reviewers will receive the points for that pull request.
+
+That said, if an issue/PR has four or more people assigned
```
Comment (elizabethli31) [inline website/project/issue-pr-grading.rst:321]: Changing language to "more than 3" instead of "four or more" would be more clear.

**Review by elizabethli31 — APPROVED** [review]
Generally looks good. Only thing I would add is to stress the number of people that can be assigned to a PR/issue. Also, is this the same number for minor issues as well?

File: website/project/issue-pr-grading.rst (hunk 2 above)
Comment (fkbad) [reply website/project/issue-pr-grading.rst:261]: I believe its that all reviewers get points and only the one TA who is that teams senior dev needs to contribute.

File: website/project/presentation.rst [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+  not as part of the ChiGame website)..
+- Your demo must involve running code you have written this quarter,
+  and that code *must* be in the ``dev`` branch of the repository.
+  You cannot demo code that is still in one of your branches and has
+  not gone through a code review. Note: The code doesn’t necessarily
+  have to be in the head of the ``dev`` branch. If someone breaks
+  your code in ``dev``, you can “rewind” to a previous commit in ``dev``
+  if you prefer.
+- To ensure there is enough time for all the teams to showcase their
+  work, your demo should be no more than ~10 minutes long.
```
Comment (fkbad) [inline website/project/presentation.rst:105]: Given we have two hours for 6 teams, is it not okay to do 15 minutes?

**Review by majorsylvie — APPROVED** [review]
Looks really good, one clarification about the length for demos (since we have the smaller class size).

Think it's good to go!

File: website/project/issue-pr-grading.rst (hunk 2 above)
Comment (borjasotomayor) [reply website/project/issue-pr-grading.rst:261]: Correct, all reviewers get points. I think for these kind of cross-component pull requests, we would only require a code review from the TA of the team that authored the PR (and only get a code review from the other TA if they actually want to be involved in the PR)

File: website/project/issue-pr-grading.rst (hunk 3 above)
Comment (borjasotomayor) [reply website/project/issue-pr-grading.rst:305]: Yes, they would receive points for both. I've added a sentence to clarify this.

File: website/project/issue-pr-grading.rst (hunk 5 above)
Comment (borjasotomayor) [reply website/project/issue-pr-grading.rst:321]: Fixed

File: website/project/presentation.rst (hunk 6 above)
Comment (borjasotomayor) [reply website/project/presentation.rst:105]: Yes, we can do 15 minutes, but if they're just going to do a demo, we'll probably tell them that they should be aiming for 10 minutes anyways.

**Pull request merged**